usr/lib/anon-shared-helper-scripts/tor_enabled_check

```diff
@@ -23,40 +23,24 @@ check_tor_enabled_do() {
       fi
    fi
 
-   local line file_name file_name_list i
-
    shopt -s globstar
    shopt -s nullglob
+   shopt -s nocasematch
 
-   if [ -f /usr/share/tor/tor-service-defaults-torrc ]; then
-      file_name_list+="/usr/share/tor/tor-service-defaults-torrc"
-      file_name_list+=" "
-   fi
-   if [ -f /etc/tor/torrc ]; then
-      file_name_list+="/etc/tor/torrc"
-      file_name_list+=" "
-   fi
+   ## many function calls in this script are provided by torrc-parser
+   source /usr/lib/anon-gw-anonymizer-config/torrc-parser
```

Comment:
This won't be available on Whonix-Workstation but I guess it's ok since above `if [ ! -e "/usr/share/anon-gw-base-files/gateway" ]; then`.

Reply:
Yes. I think so. :)